docs/getting-started/create-configuration.md

```diff
@@ -571,10 +571,10 @@ metadata:
     vpc: default
 spec:
   crossplane:
-    version: ">=v1.2.0-0"
+    version: ">=v1.4.0-0"
   dependsOn:
     - provider: crossplane/provider-aws
-      version: "v0.18.2"
+      version: ">=v0.22.0"
```

Comment:
@haarchri were you able to test out with these changes? I believe that there was an issue that required modifying the Compositions in order to upgrade to newer provider-aws versions.

Reply:
Tested clean-rollout what issue you mean ?

Reply:
@haarchri just wanted to check that everything provisioned correctly with `v0.22.0` -- it looks like we pinned to `v0.18.2` before because we just needed higher than `v0.18.1` (ref https://github.com/crossplane/crossplane/pull/2309#discussion_r705852208). In practice, we really should only need to specify the "earliest version that works" because it allows more flexibility for folks who already have `provider-aws` installed and may have constraints imposed by other packages that prevent them from upgrading. Meanwhile, folks installing fresh will still get the latest version as long as we use a `>=` constraint. Is there anything in these packages that suggest a `>=v0.18.2` constraint would not be sufficient?

Reply:
As an newbe/enduser I would expect to get the latest supported version of the provider. Furthermore as far as my knowledge goes provider-aws just supports the latest version, without major backports. At the time being thats 0.22.0. So people should also be "enforced" to use that one, so they see the and can use all features.
And there is the question of breaking changes, between versions.

Reply:
@dwerder thanks for the feedback! I respectfully disagree -- we should not enforce users to use `v0.22.0` if older versions are also compatible. New users that come along will still get the latest supported version of the provider because it will be the "latest image in the semantic version range" with a constraint like `>=v0.18.2`. We have to keep in mind that folks may have other packages that do not support newer versions of `provider-aws` and we should not force rebuild of those packages when this one would work with the `provider-aws` version they already have installed. More information on package installs [here](https://crossplane.io/docs/v1.5/concepts/packages.html#installing-a-package) :)

Reply:
@hasheddan  changed to` >=v0.18.2`